<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEntry</class>
 <widget class="QDialog" name="AddEntry">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>14</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Select what you want to add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="whichToPick">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>Scientist</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Computer</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="whichToAdd">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="comNameText">
          <property name="text">
           <string>Enter name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="comNameInput">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="comTypeText">
          <property name="text">
           <string>Enter type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="comTypeInput">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Was the computer built?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="comBuiltButton">
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="comNotBuiltButton">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="comYearBuilt">
          <property name="text">
           <string>Enter the year it was built</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="comYearBuiltInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1800</number>
          </property>
          <property name="maximum">
           <number>2015</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Enter name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sciNameInput">
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sciGender">
          <property name="text">
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="sciMaleButton">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sciFemaleButton">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="sciBirthYearText">
          <property name="text">
           <string>Enter year of birth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sciBirthInput">
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>1800</number>
          </property>
          <property name="maximum">
           <number>2015</number>
          </property>
          <property name="value">
           <number>2015</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sciAliveText">
          <property name="text">
           <string>Is the person alive?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="sciAlivButton">
            <property name="text">
             <string>Yes</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sciDeadButton">
            <property name="text">
             <string>No</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="sciDeathYearText">
          <property name="text">
           <string>Enter year of death</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sciDeathInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>2015</number>
          </property>
          <property name="value">
           <number>2015</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="addOrCancelSci">
          <property name="whatsThis">
           <string/>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Abort|QDialogButtonBox::Ok</set>
          </property>
          <property name="centerButtons">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
